c10/core/DispatchKeySet.h

```diff
@@ -108,6 +116,11 @@ class DispatchKeySet final {
     // didn't do it for now.
     return static_cast<DispatchKey>(64 - llvm::countLeadingZeros(repr_));
   }
+
+  DispatchKey highestPriorityBackendTypeId() const {
```

Comment:
Hmm, it's a bit unexpected to need this; you could unfuse the "mask with `DispatchKeySet(DispatchKeySet::FULL_AFTER_INCLUSIVE, DispatchKey::EndOfBackendKeys)`" operation. I'll keep an eye out for its usage site.